apps/fluent-tester/windows/add-win2d-dependency.js

```diff
@@ -0,0 +1,20 @@
+const fs = require('fs/promises');
+const path = require('path');
+
+/**
+ * As of this commit, the Win2D.uwp cpp package is not build transitive. Because react-native-svg consumes this as a dependency, this
+ * causes issues when using react-native-test-app to build our app. To get around this, we directly add Win2D.uwp as a dependency to
+ * ReactTestApp. This script automates this in the CI while Marlene Cota experiments with autolinking.
+ *
+ * This version of the script is also super fragile. If the pipeline fails for uwp, the contents of replacement.xml should be updated.
+ */
+
+console.warn('This script is prone to breaking. Should the UWP pipeline fail, this is the first place to check.');
+
+const replacementPath = path.resolve(__dirname, 'replacement.xml');
+const cppProjPath = path.resolve(__dirname, '..', 'node_modules', '.generated', 'windows', 'ReactTestApp', 'ReactTestApp.vcxproj');
+
+fs.readFile(replacementPath)
+  .then((contents) => fs.writeFile(cppProjPath, contents))
+  .then(() => console.log('Updated contents of ReactTestApp.vcxproj.'))
+  .catch((error) => console.log('Unable to update ReactTestApp.vcxproj: ' + error.message));
```

Comment:
Why not add this to the package.json? 
Also is this something the react-native-test-app team should know about so they can patch it on their end? (@tido64)

Reply:
Yeah I know this is super janky. If anyone else on FURN has a better idea, I'm welcome to implement it.

Reply:
Also is there a better place to put the script + xml doc in the project?

Reply:
@tido64 is this something that might be better suited to be patched in RNTA directly?

Reply:
Can't you add this as a patch using [patch-package](https://www.npmjs.com/package/patch-package) instead? It will be a lot more resilient than this. This will immediately break when someone bumps react-native-windows.

In general, we try to avoid app/library specific bits in RNTA. I want to see what @marlenecota comes up with before we consider that route.